pkg/doc/verifiable/presentation.go

```diff
@@ -313,6 +327,49 @@ func NewPresentation(vpData []byte, opts ...PresentationOpt) (*Presentation, err
 	return vp, nil
 }
 
+func decodeCredentials(rawCred interface{}, opts *presentationOpts) ([]interface{}, error) {
```

Comment:
It's the place we check for JWT signature.
In the verifiable presentation, `verifiableCredential` field could be either single string (JWT) or list of strings (several JWTs) or struct (verifiable credential) or list structs (several verifiable credentials).

Reply:
it would be good to have a doc comment defining the possible/accepted rawCred values..

Reply:
Good idea Baha! I added some comments.